### PR TITLE
Add lesson design process figures

### LINKED ISSUE #168: Add figures illustrating the lesson design process
In [the _A Lesson Design Process_ section](https://carpentries.github.io/lesson-development-training/instructor/02-lesson-design.html#a-lesson-design-process), there are TODOs for a couple of figures to illustrate the process being described.

### PR DESCRIPTION
**Pull request by tobyhodges**
Fixes #168 

These are not particular exciting, but they serve a purpose.

### CODE REVIEW

File: episodes/02-lesson-design.md [hunk 1]
```diff
@@ -55,13 +60,19 @@ we have adapted Nicholl's five phases in this training,
 to place an emphasis on assessing learning during a workshop:
 
 1. Define desired learning outcomes
-2. Decide with what activities/examples/explanations we will try to teach these skills
-3. Create assessments to determine progress towards desired outcomes
-4. Write content to lead learners from one of these assessments to the next
+2. Create assessments to determine progress towards desired outcomes
```
Comment (mkuzak) [inline episodes/02-lesson-design.md:63]:
```suggestion
2. Design assessments to determine progress towards desired outcomes
```
to match the language used in the diagram

**Review by mkuzak — CHANGES_REQUESTED** [review]
Looks good. I think those diagrams are helpful.

File: episodes/02-lesson-design.md [hunk 2]
```diff
@@ -36,8 +36,13 @@ to directly meet those stated outcomes.
...
 
-**TODO: add a figure illustrating the setps in the list above.
-could be based on Fig. 2 in [f1000-course-design-guide]**
+![Nicholl's five phases of curriculum design, 
+presented as a cycle to reflect that all aspects of the design should be 
+revisited in response to the results of the evaluation that takes place at step 5.
+](fig/nicholls-five-phases.svg){
+alt="A flow diagram presenting the Nicholl's five phases of curriculum design as a cycle."
+width="67%"
+}
```
Comment (sstevens2) [inline episodes/02-lesson-design.md:45]: Maybe a `[^1]` reference could be good here to point to the original paper?

Comment (tobyhodges) [reply episodes/02-lesson-design.md:45]: I will address this in the branch for #208

**Review by mkuzak — APPROVED** [review]

**Pull request merged**